File: testdata/MethodThrows.java
import java.text.ParseException;

class MethodThrows
{
  public void a() throws RuntimeException
  {
    System.out.println("a");
  }
  
  public String 
  b()
  throws java.io.IOException
  {
    return "b";
  }

  
  public String 
  c()
  throws java.io.IOException, RuntimeException, ParseException
  {
    return "c";
  } 
}

File: testdata/expected/valswithtypes/MethodThrows.scala
import java.text.ParseException

class MethodThrows
{
  @throws(classOf[RuntimeException])
  def a() : Unit =
  {
    System.out.println("a")
  }
  
  @throws(classOf[java.io.IOException])
  def b() : String = "b"

  
  @throws(classOf[java.io.IOException])
  @throws(classOf[RuntimeException])
  @throws(classOf[ParseException])
  def c() : String = "c" 
}